addons/spreadsheet/static/src/@types/env.d.ts

```diff
@@ -2,11 +2,10 @@ import { SpreadsheetChildEnv } from "@odoo/o-spreadsheet";
 import { Services } from "services";
 
 declare module "@spreadsheet" {
-  import { Model } from "@odoo/o-spreadsheet";
-
-  export interface SpreadsheetChildEnv extends SpreadsheetChildEnv {
-    model: OdooSpreadsheetModel;
-    services: Services;
-  }
+    import { Model } from "@odoo/o-spreadsheet";
```

Comment:
Some `.d.ts` have been prettified, not sure it's necessary

Reply:
It was intentional. Maybe it should be in a separate commit, but there's no reason for these files to not be prettified right ? We should ask for them to be included in the web tooling